### PR TITLE
Docker registry login implementation

### LINKED ISSUE #171: docker registry login command
`docker login [OPTIONS] [SERVER]`

### PR DESCRIPTION
**Pull request by thilakarajk**
Fixes #171 
- Added docker registry login command

### CODE REVIEW

File: commands/docker/docker-registry-login.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+### Login to docker registry
+
+`docker login [-u {{USERNAME}}] [-p {{PASSWORD}}] [SERVER]`
```
Comment (thilakarajk) [inline commands/docker/docker-registry-login.md]: `docker login [[-u {{USERNAME}}]] [[-p {{PASSWORD}}]] [[SERVER]]`. I feel this compromises the readability. 
So, I used the above one. Let me know which looks better.

Comment (arshadkazmi42) [reply commands/docker/docker-registry-login.md]: single brackets looks fine

File: README.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ List of some daily used commands for tech geeks.
   - [Save docker image as package](commands/docker/docker-save.md)
   - [Tail docker container logs](commands/docker/docker-logs-tail.md)
   - [Tag docker image](commands/docker/docker-tag.md)
+  - [Login to docker registry](commands/docker/docker-registry-login.md)
```
Comment (arshadkazmi42) [inline README.md]: Can you move this in alphabetical order

File: commands/docker/readme.md [hunk 3]
```diff
@@ -23,3 +23,4 @@ Daily used docker commands
 - [Save docker image as package](docker-save.md)
 - [Tail docker container logs](docker-logs-tail.md)
 - [Tag docker image](docker-tag.md)
+- [Login to docker registry](docker-registry-login.md)
```
Comment (arshadkazmi42) [inline commands/docker/readme.md]: Can you move this in alphabetical order

File: README.md (hunk 2 above)
Comment (thilakarajk) [reply README.md]: Order updated

File: commands/docker/readme.md (hunk 3 above)
Comment (thilakarajk) [reply commands/docker/readme.md]: Order updated

**Pull request merged**